src/Compilers/CSharp/Portable/Symbols/Source/SourceClonedParameterSymbol.cs

```diff
@@ -151,6 +151,11 @@ internal override bool IsCallerMemberName
             get { return _originalParam.IsCallerMemberName; }
         }
 
+        internal override AttributeAnnotations FlowAnalysisAnnotations
+        {
+            get { return _originalParam.FlowAnalysisAnnotations; }
```

Comment:
>_originalParam.FlowAnalysisAnnotations [](start = 25, length = 38)

Do we have a test covering this code path?

Reply:
Add `EnsuresNotNull_BeginInvoke` and adjusted behavior (the attributes don't help nullability analysis on BeginInvoke/EndInvoke).

---
In reply to: [187942517](https://github.com/dotnet/roslyn/pull/26656#discussion_r187942517) [](ancestors = 187942517)